js/BasicJSExamples/Addition.js

```diff
@@ -0,0 +1,107 @@
+/*Introductory function to JavaScript!
+All programming languages contain functions. They are the atomistic elements of programming.
+The first function that you will probably come accross in all languages is the "adding function".
+That is exactly the function that will be presented here, to demonstrate the (very) basic 
+uses of JavaScript.
+*/
+
+/*Commenting: Whenever you see a "//" or "/*", note that it does not contribute to the 
+functionality of the code. However, commenting is a very important part of programming,
+especially when the code becomes very complex. It gives the opportunity to the programmer
+to remind himself and tell others(like you!) what the code does, what variables define, etc.
```

Comment:
We could go further and say about how it defined a contract to developers too - referred to as an [API](https://en.wikipedia.org/wiki/Application_programming_interface)

> It gives the opportunity to future developers to understand what arguments the function can receive, and what values can be expected to be returned. This can be referred to as an Application Programming Interface, or API for short.